app/properties/[id]/page.tsx

```diff
@@ -133,7 +133,8 @@ const getCurrentOwner = cache(async (propertyId: string): Promise<landlordPublic
         .from('property_ownership')
         .select('landlord_public_profiles(*)')
         .eq('property_id', propertyId)
-        .is('ended_at', null)
+        // only get the ones that are currently the owner
+        .not('ended_at', 'is', null)
```

Comment:
I'm pretty sure this change does the exact opposite of what we want

`ended_at` should be `null` when the ownership has not ended, and the user is the current landlord.

Unless I'm wrong and this does not work as expected, you want to leave this as it is, and add a separate fetch to check for ownerships where this field is not `null` to populate a list of past landlords.

Personally I would display the other landlords either in a dropdown or link it to a separate page/modal with the full breakdown of past owners with dates, but that's up to you.